src/containers/twitter-embed/twitter.js

```diff
@@ -1,42 +1,36 @@
-import React,{Suspense,setState} from "react";
+import React, { useContext } from "react";
 import "./twitter.css";
-import Loading from "../loading/Loading";
-import {TwitterTimelineEmbed} from "react-twitter-embed";
+import { TwitterTimelineEmbed } from "react-twitter-embed";
 import { twitterDetails } from "../../portfolio";
+import { Fade } from "react-reveal";
+import StyleContext from "../../contexts/StyleContext";
+var widthScreen = window.screen.width;
 
-const renderLoader = () => <Loading />;
-const cantDisplayError = "<div class='centerContent'><h2>Can't load? Check privacy protection settings</h2></div>";
-
-function timeOut(){
-  setTimeout(function() {
-    if (!(document.getElementById('twitter').innerHTML.includes('iframe'))){
-      document.getElementById('twitter').innerHTML = cantDisplayError ;
-    }
-  },
-  10000);
-}
-var widthScreen=window.screen.width;
 export default function Twitter() {
-	if (twitterDetails.userName){
+	const { isDark } = useContext(StyleContext);
+	if (twitterDetails.userName) {
 		return (
-        <Suspense fallback={renderLoader()}>
-          <div class="tw-main-div" id="twitter">       
-                  <div className="centerContent">
-                  <TwitterTimelineEmbed
-                    sourceType="profile"
-                    screenName ={ twitterDetails.userName }
-                    options={{height:400,width: {widthScreen}}}
-                    placeholder={renderLoader()}
-                    autoHeight= {false}
-                    borderColor= "#fff"
-                    noFooter= {true}
-                    onload= {timeOut()}
-                  />
-                  </div>
-          </div>
-        </Suspense>
-  );  
-	} else {  
+			<Fade bottom duration={1000} distance="20px">
+				<div class="tw-main-div" id="twitter">
+					<div className="centerContent">
+						<TwitterTimelineEmbed
+							sourceType="profile"
+							screenName={twitterDetails.userName}
+							options={{ height: 400, width: { widthScreen } }}
+							placeholder="Can't load? Check privacy protection settings"
+							autoHeight={false}
+							key={isDark ? "1" : "2"}
+							theme={isDark ? "dark" : "light"}
+							borderColor="#fff"
+							transparent={true}
+							noScrollbar={true}
+							noFooter={true}
+						/>
+					</div>
+				</div>
+			</Fade>
+		);
+	} else {
 		return null;
 	}
 }
```

Comment:
We have removed suspense and functionality for twitter Iframe Loading. 
I'll have a look into this.